modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/ColumnParams.java

```diff
@@ -17,21 +17,29 @@
 
 package org.apache.ignite.internal.catalog.commands;
 
-import java.io.Serializable;
 import java.util.Objects;
 import org.apache.ignite.sql.ColumnType;
 
 /** Defines a particular column within table. */
-public class ColumnParams implements Serializable {
-    private static final long serialVersionUID = 5602599481844743521L;
+public class ColumnParams {
+    /** Creates parameters builder. */
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /** Column name. */
+    private String name;
 
-    private final String name;
+    /** Column type. */
+    private ColumnType type;
 
-    private final ColumnType type;
+    /** Nullability flag. */
+    private boolean nullable;
 
-    private final boolean nullable;
+    /** Column default value. */
+    private DefaultValue defaultValueDefinition;
 
-    private final DefaultValue defaultValueDefinition;
+    private ColumnParams() {}
 
     /** Creates a column definition. */
     public ColumnParams(String name, ColumnType type, DefaultValue defaultValueDefinition, boolean nullable) {
```

Comment:
It's a bit confusing to see a public constructor with parameters and a builder for the same class.